mergify_engine/actions/merge/action.py

```diff
@@ -38,6 +38,16 @@
 MARKDOWN_TITLE_RE = re.compile(r"^#+ ", re.I)
 MARKDOWN_COMMIT_MESSAGE_RE = re.compile(r"^#+ Commit Message ?:?\s*$", re.I)
 
+PRIORITY_ALIASES = {
+    "low": 0,
+    "default": 10,
```

Comment:
```suggestion
    "medium": 50,
```

Not sure why not use full mid interval?

It seems a bit small to have only 10 between low and medium while you have 90 between medium and high 🤔 

100 levels is probably enough for most people though.